From 072545d281cec5d52300c61d56ad967123282d50 Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Sat, 3 Oct 2020 12:36:40 +0530
Subject: [PATCH 1701/1852] i2c-cadence: Replace the value with enum

commit 230641c54ec01d23f6f2a0d27b7083fa5d40e13a from
https://github.com/Xilinx/linux-xlnx.git

Replace the number with an enum variable to fix the coverity warning.

Addresses-Coverity: Event mixed_enum_type.
Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/i2c/busses/i2c-cadence.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/i2c/busses/i2c-cadence.c b/drivers/i2c/busses/i2c-cadence.c
index 7cd931f85ecb..3ec81a7e75cf 100644
--- a/drivers/i2c/busses/i2c-cadence.c
+++ b/drivers/i2c/busses/i2c-cadence.c
@@ -1258,8 +1258,8 @@ static int cdns_i2c_init_recovery_info(struct cdns_i2c *pid,
 	pid->pinctrl_pins_gpio = pinctrl_lookup_state(pid->pinctrl, "gpio");
 
 	/* Fetches GPIO pins */
-	rinfo->sda_gpiod = devm_gpiod_get(&pdev->dev, "sda-gpios", 0);
-	rinfo->scl_gpiod = devm_gpiod_get(&pdev->dev, "scl-gpios", 0);
+	rinfo->sda_gpiod = devm_gpiod_get(&pdev->dev, "sda-gpios", GPIOD_ASIS);
+	rinfo->scl_gpiod = devm_gpiod_get(&pdev->dev, "scl-gpios", GPIOD_ASIS);
 
 	/* if GPIO driver isn't ready yet, deffer probe */
 	if (PTR_ERR(rinfo->sda_gpiod) == -EPROBE_DEFER ||
-- 
2.31.1

